### PR TITLE
Find by time

### PR DESCRIPTION
**Pull request by jjamay**

### CODE REVIEW

File: src/main/java/seedu/todolist/logic/parser/FindCommandParser.java [hunk 1]
```diff
@@ -19,44 +30,87 @@
...
-
-                return new FindCommand(uniqueTagList);
             }
+            return new FindCommand(keywordsSet, startTimeSet, endTimeSet, completeTimeSet, new UniqueTagList(tagsSet));
         } catch (IllegalValueException ive) {
             return new IncorrectCommand(ive.getMessage());
         }
     }
+
+    private Date parseDateParameter(String dateString) throws IllegalValueException {
```
Comment (justinhw) [inline src/main/java/seedu/todolist/logic/parser/FindCommandParser.java:96]: do we also want to include yesterday?

**Review by justinhw — APPROVED** [review]

**Comment by justinhw** [comment]
Looks good to me - can fix the codacy errors next time though

File: src/main/java/seedu/todolist/logic/parser/FindCommandParser.java (hunk 1 above)
Comment (jjamay) [reply src/main/java/seedu/todolist/logic/parser/FindCommandParser.java:96]: ill add it for next week

**Comment by jjamay** [comment]
good ol' codcay

**Pull request merged**